fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -1024,4 +1031,29 @@ private void checkAclLinkHeader(final List<String> links) throws RequestWithAclL
                     "Unable to handle request with the specified LDP-RS as the ACL.");
         }
     }
+
+    private void handleRequestDisallowedOnMemento() {
+        try {
+            addLinkAndOptionsHttpHeaders(resource());
+        } catch (Exception ex) {
+            // Catch the exception to ensure status 405 for any requests on memento.
+            LOGGER.debug("Unable to add link and options headers for PATCH request to memento path {}: {}.",
+                externalPath, ex.getMessage());
+        }
+
+        LOGGER.info("Unable to handle {} request on a path containing {}. Path was: {}", request.getMethod(),
+            FedoraTypes.FCR_VERSIONS, externalPath);
+    }
+
+    /*
+     * Ensure that an incoming versioning/memento path can be converted.
+     */
+    private void checkMementoPath() {
+        if (externalPath.contains("/" + FedoraTypes.FCR_VERSIONS)) {
+            final String path = toPath(translator(), externalPath);
+            if (path.contains(FedoraTypes.FCR_VERSIONS) && !LDPCV_TIME_MAP.equals(FedoraTypes.FCR_VERSIONS)) {
```

Comment:
`!LDPCV_TIME_MAP.equals(FedoraTypes.FCR_VERSIONS)` appears to be comparing two constants which do not have the same value, so it seems like this will always evaluate to true. Is it needed?

Reply:
Yes. This can be removed.